_maps/map_files/Pahrump/Pahrump-Underground-1.dmm

```diff
@@ -15893,7 +15901,15 @@
 /turf/open/floor/f13/wood,
 /area/f13/ncr)
 "mTe" = (
-/mob/living/simple_animal/hostile/enclave/soldier,
+/mob/living/simple_animal/hostile/enclave/soldier{
+	desc = "A former Enclave soldier, mutated by the forced evolutionary virus and grafted to the interior of his suit.";
+	extra_projectiles = 1;
+	faction = list("wastebot");
+	loot = null;
+	name = "Private Harry";
+	speak = list("AUUGHGHHH!");
+	speak_emote = list("shouts")
+	},

```

Comment:
Did you do these var-edits? This should be a custom type instead.

Reply:
I just copy-pasted what was there already. Before this chance(as it is now), there is one mob with wrong faction so they end up dying round-start.